com.unity.render-pipelines.universal/Editor/AssemblyInfo.cs

```diff
@@ -1,5 +1,7 @@
 using System.Runtime.CompilerServices;
 
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")] // permit using internal interfaces with Moq
```

Comment:
What internal API moq needs from URP?

Reply:
`AnimationClipUpgrader.IAnimationClip` and `UpgradeUtility.IMaterial`. There might be more if we wish to add new mockable interfaces to the test package in the future.